docs/website/src/teaser-page.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: teaser-page/teaser-page.njk
+eleventyExcludeFromCollections: true
+---
+<div class="ds-teaser">
+
+  <div class="ds-teaser__logo"></div>
+  
+  <div class="ds-teaser-background ds-teaser__dots"></div>
+  
+  <div class="ds-teaser-background ds-teaser__chat"></div>
+  
+  <div class="ds-teaser-background ds-teaser__arrow"></div>
+  
+  <div class="ds-teaser-background ds-teaser__user"></div>
+  
+  <div class="ds-teaser-background ds-teaser__dots-second"></div>
+  
+  <div class="ds-teaser-background ds-teaser__empty"></div>
+  
+  <div class="ds-teaser-background ds-teaser__empty-second"></div>
+  
+  <div class="ds-teaser-background ds-teaser__right-arrow"></div>
+  
+  <div class="ds-teaser-background ds-teaser__list"></div>
+  
+  <div class="ds-teaser-background ds-teaser__mobile"></div>
+  
+  <div class="ds-teaser-background ds-teaser__left-arrow"></div>
+  
+  <div class="ds-teaser-background ds-teaser__notification"></div>
+  
+  <div class="ds-teaser-background ds-teaser__message"></div>
+  
+  <div class="ds-teaser-background ds-teaser__dots-half"></div>
+  
+  <div class="ds-teaser-background ds-teaser__mobile-bottom"></div>
+  
+  <div class="ds-teaser-background ds-teaser__empty-half"></div>
+  
+  <div class="ds-teaser__wrapper">
+
+  <h1>Sanoma Learning <span class="ds-teaser__heading-part">Design System</span></h1>
+  
+  Hold tight! This documentation website is coming soon.
+  
+  <div class="ds-teaser__card-wrapper">
+  <a href="https://github.com/orgs/sanomalearning/projects/2" target="_blank" class="ds-teaser__card">
+  <div class="ds-teaser__card-text">
+    <p class="ds-teaser__card-text-heading">Planning</p>
+  <span>See more info here</span>
+  </div>
+  <div class="ds-teaser__card-icon ds-teaser__card-icon--github">
+  </div>
+  </a>
+  
+  <a href="https://sanoma.slack.com/archives/C03SA9HUUA3" class="ds-teaser__card">
+  <div class="ds-teaser__card-text">
+    <p class="ds-teaser__card-text-heading">Slack</p>
+    <span>#sl-design-system</span>
+  </div>
+  <div class="ds-teaser__card-icon ds-teaser__card-icon--slack">
+  </div>
+  </a>
+  </div>
```

Comment:
The indentation of this file is not correct? Or is that just how GitHub displays it?

Reply:
Yeah, I fixed it.